combat-trainer.lic

```diff
@@ -1077,10 +1083,13 @@ class SpellProcess
     @custom_cast = data['cast']
     @symbiosis = data['symbiosis']
     @after = data['after']
+    @skill = data['skill']
+    @prep = data['prep']
     @command = data['command']
     @reset_expire = data['expire'] ? data['abbrev'] : nil
     Flags.reset('ct-spellcast')
     @before = data['before']
+    game_state.action_taken if (@skill == 'Targeted Magic' || @prep == 'target') && game_state.weapon_skill == 'Targeted Magic'
```

Comment:
I believe part of the spell data merge logic also searches for spells listed as "sorcery" and inserts a "prep: target", so this covers the spells where we either have data on it or the user lists prep: target